browser-test/src/session_timeout.test.ts

```diff
@@ -0,0 +1,167 @@
+import {test, expect} from './support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsTestUser,
+  validateScreenshot,
+  validateToastMessage,
+} from './support'
+import {waitForHtmxReady} from './support/wait'
+import {
+  setupSessionTimeout,
+  verifyTimeoutModalBehavior,
+  verifyModalDismissal,
+  verifyAutoLogout,
+  advanceToSessionInactivityWarning,
+  advanceToSessionLengthWarning,
+  advanceToSessionTimeout,
+} from './support/session_timeout'
+
+test.describe('Session timeout warnings for Admin', () => {
+  test.beforeEach(async ({page, adminSettings}) => {
+    await setupSessionTimeout({page, adminSettings})
+    await loginAsAdmin(page)
+    await page.goto('/admin/programs')
+    await page.waitForLoadState('networkidle')
```

Comment:
```suggestion
    await waitForPageJsLoad(page)
```

(blocking) You'll be redirected to the correct page already, you just need to use our wait function. Also you should almost never need to use `networkidle` with Playwright per their docs.